src/rest/HTTPError.js

```diff
@@ -42,8 +42,9 @@ class HTTPError extends Error {
     /**
      * The attachment data that is sent to Discord
      * @typedef {Object} HTTPAttachmentData
+     * @property {string} attachment The attachment
```

Comment:
Probably should have a better description, wasn't sure what to put 😅.

Reply:
It looks like that's the source for the attachment data (so it could be a stream or buffer too).
So maybe something like this? (Would also need updating in the typings again 😅 )
```suggestion
     * @property {string|Buffer|Stream} attachment The source of this attachment data
```

---

Assigned from `fileLike` (what was passed as file to djs to send)
https://github.com/discordjs/discord.js/blob/99e2ca408b0ead894c2d91bc2902c392a3635fb0/src/structures/APIMessage.js#L295-L298

Reply:
Done 😄